src/libs/actions/Report.js

```diff
@@ -847,6 +880,7 @@ function deleteReportComment(reportID, reportAction) {
                 },
             },
         },
+        reportOnyxUpdates.failureUpdate,
```

Comment:
We were not reverting the changes to the report if the `DeleteComment` command failed

Reply:
Before 🐛 :



https://github.com/Expensify/App/assets/87341702/6a0eae95-6a02-4c8f-90dd-c246eab05287


After [FIXED]:

https://github.com/Expensify/App/assets/87341702/cfb8f9a0-264c-44a3-9669-96fce225e6b1